docs/src/pages/components/switchfield/examples/SwitchFieldThumbColorExample.tsx

```diff
@@ -5,7 +5,7 @@ export const SwitchFieldThumbColorExample = () => {
   return (
     <SwitchField
       label="This is a switch"
-      thumbColor={`${tokens.colors.red[60]}`}
+      thumbColor={`${tokens.colors.orange[10]}`}
```

Comment:
This color make sure 3:1 contrast ratio in both light and dark mode

Reply:
Could we make this un stringified? This line could be simplified to `thumbColor={tokens.colors.orange[10]}`